SYCL/Basic/accessor/accessor.cpp

```diff
@@ -60,6 +60,57 @@ template <typename Acc> struct Wrapper2 {
 
 template <typename Acc> struct Wrapper3 { Wrapper2<Acc> w2; };
 
+template <typename T> void TestAccSizeFuncs(const std::vector<T> &vec) {
+  std::vector<size_t> res(vec.size()), locRes(vec.size());
+  try {
+    sycl::queue q;
+    sycl::buffer<T> buf(vec.data(), vec.size());
+    sycl::buffer<size_t> bufRes(res.data(), res.size());
+    sycl::buffer<size_t> bufLocRes(locRes.data(), locRes.size());
+
+    q.submit([&](sycl::handler &cgh) {
+      auto acc = buf.template get_access<sycl::access::mode::read>(cgh);
+      auto accRes = bufRes.get_access<sycl::access::mode::write>(cgh);
+      auto accLocRes = bufLocRes.get_access<sycl::access::mode::write>(cgh);
+      sycl::local_accessor<T, 1> localAcc(vec.size(), cgh);
+      cgh.single_task([=]() {
+        accRes[0] = acc.byte_size();
+        accRes[1] = acc.size();
+        accRes[2] = acc.max_size();
+        accRes[3] = size_t(acc.empty());
+        accLocRes[0] = localAcc.byte_size();
+        accLocRes[1] = localAcc.size();
+        accLocRes[2] = localAcc.max_size();
+        accLocRes[3] = size_t(localAcc.empty());
+      });
+    });
+    q.wait();
+
+    auto acc = buf.template get_access<sycl::access::mode::read>();
+    assert(acc.byte_size() == vec.size() * sizeof(T));
+    assert(acc.size() == vec.size());
+    assert(acc.max_size() ==
+           std::numeric_limits<
+               typename sycl::host_accessor<T>::difference_type>::max());
+    assert(acc.empty() == vec.empty());
+  } catch (sycl::exception &e) {
+    std::cout << e.what() << std::endl;
+  }
```

Comment:
Why do we need that? Wouldn't it be fine to automatically `std::terminate` ? Why are we ignoring the exception?

Reply:
Deleted [97607ba](https://github.com/intel/llvm-test-suite/pull/1265/commits/97607baf3d2b37a18bbae1633c990df0d33d74d0)